www/_template/reference/configuration.md

```diff
@@ -124,211 +337,239 @@ module.exports = {
+Configure your tests
 
-```diff
--import '../../../../../Button.js';
-+import '@app/Button.js';
-```
+TODO: is this enough? should we be more specific like "how Snowpack runs tests?"
+
+### testOptions.files
+
+**Type**: `string[]`
+
+**Default**: `["__tests__/**/*", "**/*.@(spec|test).*"]`
+
+Specifies what files Snowpack recognizes as tests.
+
+All matching test files are scanned for installable dependencies during development, but excluded from both scanning and building in your final build.
+
+TODO: I think this is using glob matching?
+## config.experiments
+
+**Type**: `object` (option name: value)
+
+This section is experimental and not yet finalized. May change across versions.
+
+[See the code for options](https://github.com/snowpackjs/snowpack/blob/main/snowpack/src/types/snowpack.ts#L235)
+
+## config.\_extensionMap
+
+TODO: I think this matches input extentions to output but I am not sure
+
+## config.proxy
+
+TODO: I no longer see this in the types.ts file is it deprecated?
+
+**Default**: TODO
+
+Configure the dev server to proxy requests. See the section below for all options.
+
+If desired, `"proxy"` is where you configure the proxy behavior of your dev server. Define different paths that should be proxied, and where they should be proxied to.
+
+The short form of a full URL string is enough for general use. For advanced configuration, you can use the object format to set all options supported by [http-proxy](https://github.com/http-party/node-http-proxy).
+
+This configuration has no effect on the final build.
 
 Example:
 
 ```js
 // snowpack.config.json
 {
-  alias: {
-    // Type 1: Package Import Alias
-    "lodash": "lodash-es",
-    "react": "preact/compat",
-    // Type 2: Local Directory Import Alias (relative to cwd)
-    "components": "./src/components"
-    "@app": "./src"
+  "proxy": {
+    // Short form:
+    "/api/01": "https://pokeapi.co/api/v2/",
+    // Long form:
+    "/api/02": {
+      on: { proxyReq: (p, req, res) => /* Custom event handlers (JS only) */ },
+      /* Custom http-proxy options */
+    }
   }
 }
 ```
 
-### config.exclude
+Options:
 
-Exclude will ignore certain directories from being watched. Example:
+- **`"path".on`** | `object` (string: function)
+  - `on` is a special Snowpack property for setting event handler functions on proxy server events. See the section on ["Listening for Proxy Events"](https://github.com/http-party/node-http-proxy#listening-for-proxy-events) for a list of all supported events. You must be using a `snowpack.config.js` JavaScript configuration file to set this.
+- All options supported by [http-proxy](https://github.com/http-party/node-http-proxy).
 
-```js
-// snowpack.config.json
-{
-  exclude: ['**/vendor/**/*'];
-}
-```
+## Environment Variables
 
-### Environment Variables
+TODO: Figure out where this goes
```

Comment:
@FredKSchott this section doesn't seem to belong here, should it get its own page?

Reply:
Yea, although not sure where. I think giving it it's own top-level reference page is fine for now.